migration/docker/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM --platform=linux/amd64 eclipse-temurin:21
+ARG KAFKA_VERSION="3.4.0"
+ARG KAFKA_CONNECT_VERSION="${KAFKA_VERSION}"
+ARG KAFKA_SCALA_VERSION="2.13"
+ARG PUBSUB_GROUP_KAFKA_CONNECTOR_VERSION="1.2.0"
+ARG KAFKA_HOME_ROOT="/opt"
+ENV KAFKA_HOME="${KAFKA_HOME_ROOT}/kafka"
+ARG KAFKA_CONFIG_DIR="${KAFKA_HOME}/config"
+ARG KAFKA_RELEASE="kafka_${KAFKA_SCALA_VERSION}-${KAFKA_VERSION}"
```

Comment:
why is KAFKA_SCALA_VERSION included in the KAFKA_RELEASE?

Reply:
Most of these came pre-configured with the work Pramod did. I'm not entirely sure here